app/uk/gov/hmrc/customs/rosmfrontend/controllers/ApplicationController.scala

```diff
@@ -32,23 +33,32 @@ import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
 class ApplicationController @Inject()(
-  override val currentApp: Application,
-  override val authConnector: AuthConnector,
-  mcc: MessagesControllerComponents,
-  viewStart: start,
-  migrationStart: migration_start,
-  accessibilityStatementView: accessibility_statement,
-  cdsFrontendDataCache: SessionCache,
-  appConfig: AppConfig
-)(implicit override val messagesApi: MessagesApi, ec: ExecutionContext)
-    extends CdsController(mcc) {
+                                       override val currentApp: Application,
+                                       override val authConnector: AuthConnector,
+                                       mcc: MessagesControllerComponents,
+                                       viewStart: start,
+                                       migrationStart: migration_start,
+                                       accessibilityStatementView: accessibility_statement,
+                                       cdsFrontendDataCache: SessionCache,
+                                       appConfig: AppConfig
+                                     )(implicit override val messagesApi: MessagesApi, ec: ExecutionContext)
```

Comment:
Prefer the previous style, but it's just a formatting.

@SteveSugden What do you think about introducing scalafmt? We could do this after we merge PRs in progress.

Reply:
Good idea.  This formatting was (I'm guessing) the built-in IDEA